src/Magnum/ImGuiIntegration/Integration.h

```diff
@@ -57,11 +57,27 @@ Example usage:
 #include "Magnum/ImGuiIntegration/visibility.h" /* defines IMGUI_API */
 
 #include <imgui.h>
+#include <Corrade/Containers/StringView.h>
```

Comment:
This would ideally be also wrapped in `#ifdef IMGUI_HAS_IMSTR`. 

Alternatively I was thinking of deinlining the conversion into a new `Integration.cpp` file and having just a forward declaration here, but that's maybe taking it a bit too far. The `StringView` header is not that huge.